docs/ui/insight/get-started.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Get started with RedisInsight
+linkTitle: Get started
+description: Get started with RedisInsight
+weight: 20
+
+---
+
+RedisInsight is available as a standalone download or [bundled with Redis Stack](/docs/stack/get-started/install). Install it and try the in-app examples available in Workbench. For more information, see [Examples](/docs/ui/insight/examples).
+
+You can use RedisInsight on Windows, MacOS, or Linux.
+
+## Install on Windows
+
+### From website
+
+1. Download [RedisInsight](https://redis.com/redis-enterprise/redis-insight). 
+
+2. Open the `RedisInsight-v2-win-installer.exe` file.
+
+3. Open the RedisInsight icon.
+
+### From stores
+
+Go to [Microsoft Windows Apps](https://apps.microsoft.com/store/detail/redisinsight/XP8K1GHCB0F1R2) and download the RedisInsight app.
+
+To resume your journey, [start Redis Stack Server](#start-redis-stack-server).   
+
+## Install on MacOS
+
+You can install RedisInsight for MacOS from the Redis website or Homebrew. (Redis plans to release the RedisInsight app in Apple Store.)
```

Comment:
we released on the Apple Store -  https://apps.apple.com/us/app/redisinsight/id6446987963

Reply:
i'll update!